pkg/kafka/auth/options.go

```diff
@@ -26,22 +26,24 @@ const (
 	defaultAuthentication = none
 
 	// Kerberos configuration options
-	kerberosPrefix            = ".kerberos"
-	suffixKerberosServiceName = ".service-name"
-	suffixKerberosRealm       = ".realm"
-	suffixKerberosUseKeyTab   = ".use-keytab"
-	suffixKerberosUsername    = ".username"
-	suffixKerberosPassword    = ".password"
-	suffixKerberosConfig      = ".config-file"
-	suffixKerberosKeyTab      = ".keytab-file"
+	kerberosPrefix                = ".kerberos"
+	suffixKerberosServiceName     = ".service-name"
+	suffixKerberosRealm           = ".realm"
+	suffixKerberosUseKeyTab       = ".use-keytab"
+	suffixKerberosUsername        = ".username"
+	suffixKerberosPassword        = ".password"
+	suffixKerberosConfig          = ".config-file"
+	suffixKerberosKeyTab          = ".keytab-file"
+	suffixKerberosDisablePAFXFAST = ".disable-pa-fx-fast"
```

Comment:
in the description you refer to this as "FAST negotiation". Is it important to have "pa-fx"? If so, it would be better to spell them out in the flag description too.